Makefile

```diff
@@ -58,3 +58,18 @@ end-to-end-test:
 	    -v `pwd`/.nyc_output/:/tmp -v `pwd`:/mddb -e MDDB='/mddb/metadata.db' \
 	    welder/web-e2e-tests:latest \
 	    xvfb-run -a -s '-screen 0 1024x768x24' npm run test -- --verbose
+
+cockpit-test:
+	sudo docker network create welder
+	sudo docker build -f Dockerfile.cockpit --cache-from welder/web-cockpit:latest -t welder/web-cockpit:latest .
+	sudo docker build -f ./test/end-to-end/Dockerfile --cache-from welder/web-e2e-tests:latest -t welder/web-e2e-tests:latest ./test/end-to-end/
+
+	wget --progress=dot:giga https://s3.amazonaws.com/weldr/metadata.db
+	sudo docker run -d --name api --restart=always -p 4000:4000 -v bdcs-recipes-volume:/bdcs-recipes -v `pwd`:/mddb --network welder --security-opt label=disable welder/bdcs-api-rs:latest
+	sudo docker run -d --name web --restart=always -p 9090:9090 -v `pwd`/public:/usr/share/cockpit/welder --network welder welder/web-cockpit:latest
```

Comment:
The above few lines are very similar to what the `ent-to-end` test target does and I don't like the repetition of setup steps. Instead can we move the setting up of api container, download of metadata, etc (all common steps) into a separate make target and have both e2e and cockpit tests depend on it. Then only setup whatever specific contaienrs are needed.

Reply:
Also I think `docker run --rm` for web and api containers will not hurt here and better keep the test environment clean.

Reply:
If you use -d with --rm, the container is removed when it exits or when the daemon exits, whichever happens first. In this case, both of api and web will not exit during testing.